core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -59,7 +59,7 @@
  */
 public class ContentFormatters implements FormatterRegistry {
 
-  public static final int MAX_ALT_TEXT_LENGTH = 1000;
+  public static final int MAX_ALT_TEXT_LENGTH = 250;
```

Comment:
I'd mention in a comment that the new value is coming from https://code.squarespace.net/projects/V6/repos/cms-legacy-data/browse/src/main/java/com/squarespace/v6/utils/enums/AltText.java?at=4a212f6f7664ca410c22c4c9151078e38cedb26f#13, but I'm not sure if we're okay with such comments in a public repo?